NexGenUA/nodejs-aws-be
# Task 9 "Backend For Frontend"

### [Link to task-9](https://github.com/rolling-scopes-school/nodejs-aws-tasks/edit/main/task9-bff/task.md)

## Evaluation criteria (each mark includes previous mark criteria)


### Example:

---
  - Original api cart: http://nexgenua-cart-api-dev.eu-west-1.elasticbeanstalk.com/api/profile/cart
  - Api cart from bff: http://nexgenua-bff-service-develop.eu-west-1.elasticbeanstalk.com/api/profile/cart
---
  - Original api products: https://3k7mia24j3.execute-api.eu-west-1.amazonaws.com/dev/products
  - Api products from bff: http://nexgenua-bff-service-develop.eu-west-1.elasticbeanstalk.com/products
---

### Link to bff-service:
  - HTTP link: http://nexgenua-bff-service-develop.eu-west-1.elasticbeanstalk.com/ping

You can use frontend application for test:

How to test:
  * Get products list -  https://d2c750wv33qoc2.cloudfront.net/
  * Create new product* - https://d2c750wv33qoc2.cloudfront.net/admin/product-form/
  * Get products list - result shouldn’t have new product
  * Wait more than 2 minutes
  * Get products list - result should have new product

  - https://github.com/NexGenUA/nodejs-aws-be/tree/task-9/bff-service
